modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm

```diff
@@ -1650,8 +1662,8 @@ sub hgvs_protein {
   my $ref = $tv->get_reference_TranscriptVariationAllele;
 
   ## Incase the user wants shifted HGVS but not shifted consequences, we run the shifting method  
-  my $hash_already_defined = defined($ref->{shift_hash});
-  $ref->_return_3prime(1) unless $hash_already_defined;
+  my $ref_hash_already_defined = defined($ref->{shift_hash});
```

Comment:
Could you please add a few comments why the hgvs_protein method needs updating? The main PR mentiones HGVS_transcript only. Thanks.